interpreter/cling/lib/Interpreter/IncrementalExecutor.cpp

```diff
@@ -72,11 +72,11 @@ CreateHostTargetMachine(const clang::CompilerInstance& CI) {
   std::string MCPU;
   std::string FeaturesStr;
 
-  return std::unique_ptr<TargetMachine>(TheTarget->createTargetMachine(Triple,
-                                        MCPU, FeaturesStr,
-                                        llvm::TargetOptions(),
-                                        Optional<Reloc::Model>(), CMModel,
-                                        OptLevel));
+  auto TM = std::unique_ptr<TargetMachine>(TheTarget->createTargetMachine(
```

Comment:
Could you please split formatting and functional change into two commits? I'm terrible at filtering white-space parts of diffs from functional ones... Thanks, much appreciated!

Reply:
Alright, done.